src/components/layout/ZooFooter.jsx

```diff
@@ -35,48 +34,53 @@ const ZooFooter = (props) => {
-        <Menu className="footer__nav" direction="row">
+        {props.homeLogoDiv}
+        <Menu className="footer__nav" direction="row" inline={true} responsive={false}>
           {props.socialNavList.map((navItem, i) => createKeyedAnchorItem(navItem, i))}
         </Menu>
       </Section>
       <Section pad={{ horizontal: 'large', vertical: 'none' }} full="horizontal">
         <hr className="footer__divider" />
       </Section>
-      <Section direction="row" className="footer__section" pad={{ horizontal: 'large', vertical: 'medium' }}>
+      <Section direction="row" className="footer__section" pad={{ horizontal: 'large', vertical: 'medium' }} margin={{ top: 'none', bottom: 'large', left: 'none', right: 'none' }}>
         <Columns maxCount={6} masonry={true} size="small" >
-          <Menu className="footer__nav">
+          <Menu className="footer__nav" margin={{ horizontal: 'none', vertical: 'small' }}>
             {props.projectNavList.map((navItem, i) => createKeyedAnchorItem(navItem, i))}
           </Menu>
-          <Menu className="footer__nav">
+          <Menu className="footer__nav" margin={{ horizontal: 'none', vertical: 'small' }}>
             {props.aboutNavList.map((navItem, i) => createKeyedAnchorItem(navItem, i))}
           </Menu>
-          <Menu className="footer__nav">
+          <Menu className="footer__nav" margin={{ horizontal: 'none', vertical: 'small' }}>
             {props.getInvolvedNavList.map((navItem, i) => createKeyedAnchorItem(navItem, i))}
           </Menu>
-          <Menu className="footer__nav">
+          <Menu className="footer__nav" margin={{ horizontal: 'none', vertical: 'small' }}>
             {props.talkNavList.map((navItem, i) => createKeyedAnchorItem(navItem, i))}
           </Menu>
-          <Menu className="footer__nav">
+          <Menu className="footer__nav" margin={{ horizontal: 'none', vertical: 'small' }}>
             {props.buildNavList.map((navItem, i) => createKeyedAnchorItem(navItem, i))}
           </Menu>
-          <Menu className="footer__nav">
+          <Menu className="footer__nav" margin={{ horizontal: 'none', vertical: 'small' }}>
             {props.newsNavList.map((navItem, i) => createKeyedAnchorItem(navItem, i))}
           </Menu>
         </Columns>
       </Section>
       <Section
+        align="center"
         className="footer__section--small"
         colorIndex={props.smallSectionColorIndex}
+        direction="row"
         full="horizontal"
         pad={{ horizontal: 'large', vertical: 'none' }}
         justify="between"
       >
-        <Menu className="footer__nav" direction="row" inline={true}>
+        <Menu className="footer__nav" direction="row" inline={true} responsive={false}>
           {props.policyNavList.map((navItem, i) => createKeyedAnchorItem(navItem, i))}
         </Menu>
-        {props.adminContainer &&
-          props.adminContainer}
+        <div>
+          {props.adminContainer &&
+            props.adminContainer}
```

Comment:
This is first checking for its existence and then renders it if it exists since that prop type is set to be a node. I could make this more explicit with a double bang for clarity:
```
{!!props.adminContainer &&
  props.adminContainer
```
Let me know if I should change it.